sections/gov/components/Proposal/DilutionContent.tsx

```diff
@@ -1,9 +1,13 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
```

Comment:
I dont think we need dilution anymore

Reply:
or rename this file, pretty sure we have one called "Content.tsx" already

Reply:
Yea should rename. We actually removed Content.tsx, since it was only used for the election table that no longer exists